ci/cached-builds/gha_lvm_overlay.sh

```
@@ -0,0 +1,78 @@
+sudo swapoff -a
+sudo rm -f /mnt/swapfile
+
+echo "Creating LVM Volume."
+echo "  Creating LVM PV on root fs."
+# create loop pv image on root fs
+ROOT_RESERVE_KB=$(expr ${root_reserve_mb} \* 1024)
+ROOT_FREE_KB=$(df --block-size=1024 --output=avail / | tail -1)
+ROOT_LVM_SIZE_KB=$(expr $ROOT_FREE_KB - $ROOT_RESERVE_KB)
+ROOT_LVM_SIZE_BYTES=$(expr $ROOT_LVM_SIZE_KB \* 1024)
+sudo touch "${pv_loop_path}" && sudo fallocate -z -l "${ROOT_LVM_SIZE_BYTES}" "${pv_loop_path}"
+export ROOT_LOOP_DEV=$(sudo losetup --find --show "${pv_loop_path}")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)